lizmap/www/assets/js/edition.js

```diff
@@ -1377,33 +1377,38 @@ OpenLayers.Geometry.pointOnSegment = function(point, segment) {
             // Manage child form
             if ( editionLayer['parent'] != null ){
                 var parentInfo = editionLayer['parent'];
-                var relationRefField = parentInfo['relation'].referencingField;
-                var parentFeatProp = parentInfo['feature'].properties[relationRefField];
+                var relation = parentInfo['relation'];
+                var relationRefField = relation.referencingField;
+                var parentFeatProp = parentInfo['feature'].properties[relation.referencedField];
```

Comment:
sorry, I don't see why you made this change. What you write do exactly the same thing that the old code, with more code. don't you? Did I miss something?

Reply:
You have probably missed the difference between `referencingField` and `referencedField`.

* `referencingField` is the field's name in the child layer
* `referencedField` is the field's name in the parent layer

Here the `referencingField` is used to get the form control and the `referencedField` to get the value to link child to the parent.

Reply:
@laurentj Do you approve my changes ?